Description: Rename "pump" command to "distcc-pump"
 Distcc pump mode is started by a command that upstream calls "pump",
 which is already in use by another Debian package.
 .
 We rename that command to "distcc-pump" and change any references to
 it in the rest of the package.  This patch updates the references,
 the script and it's man page are renamed in debian/rules.
Forwarded: not-needed
Bug-Debian: http://bugs.debian.org/559196
Bug-Debian: http://bugs.debian.org/594083
Author: Daniel Hartwig <mandyke@gmail.com>

--- a/source/man/distcc.1
+++ b/source/man/distcc.1
@@ -134,7 +134,7 @@
 Wrap your build inside the pump command, here assuming 10 servers:
 
 .RS
-$ pump make -j20 CC=distcc
+$ distcc-pump make -j20 CC=distcc
 .RE
 .SH "HOW PLAIN (NON-PUMP) DISTCC WORKS"
 distcc only ever runs the compiler and assembler remotely.  With plain distcc,
@@ -171,7 +171,7 @@
 corresponds to the current working directory on the client.  To find and
 transmit the many hundreds of files that are often part of a single compilation,
 pump mode uses an incremental include analysis algorithm.  The include server is
-a Python program that implements this algorithm.  The pump command starts the
+a Python program that implements this algorithm.  The distcc-pump command starts the
 include server so that throughout the build it can answer include queries by
 distcc commands.
 
@@ -534,7 +534,7 @@
 .TP
 .B ,cpp
 Enables distcc-pump mode for this host.  Note: the build command must be 
-wrapped in the pump script in order to start the include server.
+wrapped in the distcc-pump script in order to start the include server.
 .TP
 .B --randomize
 Randomize the order of the host list before execution.
@@ -884,7 +884,7 @@
 distcc was written by Martin Pool <mbp@sourcefrog.net>, with the
 co-operation of many scholars including Wayne Davison, Frerich Raabe,
 Dimitri Papadopoulos and others noted in the NEWS file.  Please report
-bugs to <distcc@lists.samba.org>.  See \fBpump\fR(1) for the authors of pump mode.
+bugs to <distcc@lists.samba.org>.  See \fBdistcc-pump\fR(1) for the authors of pump mode.
 .SH "LICENCE"
 You are free to use distcc.  distcc (including this manual) may be
 copied, modified or distributed only under the terms of the GNU
@@ -892,7 +892,7 @@
 absolutely no warrany.  A copy of the GPL is included in the file
 COPYING.
 .SH "SEE ALSO"
-\fBdistccd\fR(1), \fBpump\fR(1), \fBinclude_server\fR(1), \fBgcc\fR(1),
+\fBdistccd\fR(1), \fBdistcc-pump\fR(1), \fBinclude_server\fR(1), \fBgcc\fR(1),
 \fBmake\fR(1), and  \fBccache\fR(1).
 .I http://code.google.com/p/distcc/
 .I http://ccache.samba.org/
--- a/source/man/pump.1
+++ b/source/man/pump.1
@@ -1,23 +1,23 @@
 .TH pump 1 "9 June 2008"
 .SH "NAME"
-pump \- accelerate remote compilation with distcc
+distcc-pump \- accelerate remote compilation with distcc
 .SH "SYNOPSIS"
-.B pump
+.B distcc-pump
 .I BUILD-COMMAND [ARGS]
 \& ...
 .BR
 .PP
 eval `
-.B pump
+.B distcc-pump
 --startup `;
 .I BUILD-COMMAND [ARGS]
 \& ...
 ;
-.B pump
+.B distcc-pump
 --shutdown
 .BR
 .SH "DESCRIPTION"
-.B pump
+.B distcc-pump
 is part of distcc.
 It is used for distcc's pump mode.
 Distcc's pump mode accelerates remote compilation with distcc
@@ -36,7 +36,7 @@
 .PP
 The simplest usage is the form
 .RS
-.B pump
+.B distcc-pump
 .I COMMAND [ARG]
 \& ...
 .RE
@@ -45,7 +45,7 @@
 .B lsdistcc
 to compute the distcc host list;
 set some auxiliary environment variables;
-change PATH to use the distcc client in the same directory as the "pump"
+change PATH to use the distcc client in the same directory as the "distcc-pump"
 script;
 execute
 .I COMMAND
@@ -59,23 +59,23 @@
 "make -j80", that will do many concurrent invocations of distcc.
 .PP
 An alternative way of invoking
-.B pump
-is to explicitly invoke "pump --startup"
-to start the include server and "pump --shutdown" to stop the include server.
-The "pump --startup" command will start up the include server, and will print
+.B distcc-pump
+is to explicitly invoke "distcc-pump --startup"
+to start the include server and "distcc-pump --shutdown" to stop the include server.
+The "distcc-pump --startup" command will start up the include server, and will print
 out some environment variable settings.  These environment variables are used
 to communicate between the pump-mode "distcc" client and the include
-server, and to communicate between "pump --startup" and "pump --shutdown".
-The caller of "pump --startup" is responsible for setting those environment
-variables before invoking "distcc" or "pump --shutdown".
+server, and to communicate between "distcc-pump --startup" and "distcc-pump --shutdown".
+The caller of "distcc-pump --startup" is responsible for setting those environment
+variables before invoking "distcc" or "distcc-pump --shutdown".
 .PP
 For example:
 .RS
-eval `pump --startup`
+eval `distcc-pump --startup`
 .br
 make -j80
 .br
-pump --shutdown
+distcc-pump --shutdown
 .RE
 .PP
 Note that distcc's pump-mode assumes that sources files will not be
@@ -84,13 +84,13 @@
 may cause inconsistent results.
 .SH "INVOKING LSDISTCC"
 When invoked in either the simple "
-.B pump
+.B distcc-pump
 .I COMMAND [ARG]
 \&... " form,
 or as "
-.B pump --startup
+.B distcc-pump --startup
 \&", the
-.B pump
+.B distcc-pump
 script will invoke
 .B lsdistcc
 \& whenever DISTCC_POTENTIAL_HOSTS is set and DISTCC_HOSTS isn't.
@@ -109,12 +109,12 @@
 needed for
 .BR distcc(1)
 or
-.B pump --shutdown
+.B distcc-pump --shutdown
 to access it.
 .TP
 .B --shutdown
 Shuts down an include server started up by
-.B pump --startup.
+.B distcc-pump --startup.
 .SH "ENVIRONMENT VARIABLES"
 The following environment variables are all optional.
 .TP
@@ -154,7 +154,7 @@
 PYTHONOPTIMIZE
 If set to "", then Python optimization is disabled.
 .SH "EXAMPLE"
-.B pump make -j20
+.B distcc-pump make -j20
 .SH "BUGS"
 .\" TODO:
 .\" Fix inconsistency between BUGS section and bug reporting instructions
@@ -163,11 +163,11 @@
 .I reporting-bugs.txt
 in the documentation directory for information on how to report it.
 .SH "AUTHORS"
-The pump script and distcc's pump mode were written by Nils Klarlund,
+The distcc-pump script and distcc's pump mode were written by Nils Klarlund,
 Manos Renieris, Fergus Henderson, and Craig Silverstein. Please report
 bugs to <distcc@lists.samba.org>.
 .SH "LICENCE"
-.B pump
+.B distcc-pump
 is part of distcc.
 You are free to use distcc.  distcc (including this manual) may be
 copied, modified or distributed only under the terms of the GNU
--- a/source/man/distccd.1
+++ b/source/man/distccd.1
@@ -279,7 +279,7 @@
 Directory for temporary files such as preprocessor output.  By default
 /tmp/ is used.
 .SH "SEE ALSO"
-\fBdistcc\fR(1), \fBpump\fR(1), \fBinclude_server\fR(1), \fBgcc\fR(1),
+\fBdistcc\fR(1), \fBdistcc-pump\fR(1), \fBinclude_server\fR(1), \fBgcc\fR(1),
 \fBmake\fR(1), and  \fBccache\fR(1)
 .I http://code.google.com/p/distcc/
 .SH "BUGS"
@@ -302,5 +302,5 @@
 distcc was written by Martin Pool <mbp@sourcefrog.net>, with the
 co-operation of many scholars including Wayne Davison, Frerich Raabe,
 Dimitri Papadopoulos and others noted in the NEWS file. 
-See \fBpump\fR(1) for the authors of pump mode.
+See \fBdistcc-pump\fR(1) for the authors of pump mode.
 Please report bugs to <distcc@lists.samba.org>.
--- a/source/man/include_server.1
+++ b/source/man/include_server.1
@@ -14,7 +14,7 @@
 command itself terminates as soon as the include server has been spawned.
 .PP
 The INCLUDE_SERVER_PORT argument is the name of a socket used for all
-communication between distcc clients and the include server.  The \fBpump\fR(1)
+communication between distcc clients and the include server.  The \fBdistcc-pump\fR(1)
 command is responsible for creating the socket location, for passing it to this
 script, and for passing it to all distcc clients via the environment variable
 named INCLUDE_SERVER_PORT. 
@@ -71,7 +71,7 @@
 .TP
 .B --pid_file FILEPATH
 The pid of the include server is written to file FILEPATH. This allows a script
-such a \fBpump\fR to tear down the include server.
+such a \fBdistcc-pump\fR to tear down the include server.
 .TP                             
 .B -s, --statistics
 Print information to stdout about include analysis.
@@ -81,7 +81,7 @@
 filepath comes in or out of existence.  LIST is a colon separated string of
 filepaths, possibly containing simple globs (as allowed by Python's glob
 module). Print a warning whenever such a change happens (if warnings are
-enabled). This option allows limited exceptions to distcc_pump's normal
+enabled). This option allows limited exceptions to distcc-pump's normal
 assumption that source files are not modified during the build.
 .TP
 .B -t, --time  
@@ -147,7 +147,7 @@
 client cannot use pump-mode.
 
 To overcome this problem in a not always reliable way, set the environment
-variable INCLUDE_SERVER_ARGS='--unsafe_absolute_includes' when invoking the pump
+variable INCLUDE_SERVER_ARGS='--unsafe_absolute_includes' when invoking the distcc-pump
 script to pass the --unsafe_absolute_includes option to the include server.
 .TP
 .B Warning: Absolute filepath ... was IGNORED
@@ -185,7 +185,7 @@
 .B See the Distcc Discrepancy Symptoms section in the include_server(1) man 
 .B page.
 
-The pump script issues this message at the end of the build. This means that for
+The distcc-pump script issues this message at the end of the build. This means that for
 at least one distcc invocation a local compilation succeeded after the remote
 compilation failed. Each distcc invocation for which such a discrepancy occurred
 in turn also issues a message such as:
--- a/source/include_server/run.py
+++ b/source/include_server/run.py
@@ -56,7 +56,7 @@
   ./run.py `which pychecker` include_server.py  
 
 In installed distcc-pump:
-   # See 'pump' script.
+   # See 'distcc-pump' script.
    $include_server_location/run.py --run_in_install include_server.py .. 
 """
 
--- a/source/src/include_server_if.c
+++ b/source/src/include_server_if.c
@@ -78,7 +78,7 @@
     include_server_port = getenv("INCLUDE_SERVER_PORT");
     if (include_server_port == NULL) {
         rs_log_warning("INCLUDE_SERVER_PORT not set - "
-                       "did you forget to run under 'pump'?");
+                       "did you forget to run under 'distcc-pump'?");
         return 1;
     }
 
